__tests__/utils/operations.js

```diff
@@ -814,4 +814,23 @@ module.exports = {
     }
     return secret
   },
+  async downloadFile(config, user, recordId) {
```

Comment:
This is definitely not going to work in a browser context. We'll need to think more about how to best test the file features. I think there are ways, but we're gonna have to potentially do some environment detection and drop to some lower level primitives when reading/writing files in the test environment context.

(basically I'm thinking we send around strings, then we create buffer objects for creation, or convert from the buffer object when reading and send back the file contents. I can help with this, but it will take some work.